### ISSUE TITLE
Backport: ZFS not in path rancher-agent and rancherOS

### ISSUE THREAD

**Issue by alena1108** [rancher/rke#1194]
Backport https://github.com/rancher/rke/issues/979

**Comment by deniseschannon** [comment]
We will not plan on backporting this to a v0.2.x release.

**Comment by sangeethah** [comment]
Can be tested through Rancher rancher/rancher:v2.2.5-rc3. Docker ssh to any of the k8s system containers and check if zfs-utils tool is present.

**Comment by jiaqiluo** [comment]
The bug fix is validated on rancher: v2.2-head b240bd20

Steps:
- run rancher: v2.2-head container
- add a single-node custom cluster using the default k8s version: v1.1.4.3-rancher1-1
- after the cluster is active, ssh into the node 
- `docker ps | grep hyperkube` to find containers using the `rancher/hyperkube` image
```
$ docker ps | grep hyperkube
b6f1a7d7f717        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kube-proxy
2765534cf235        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kubelet
53dc8b45fae5        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kube-scheduler
5042e995d824        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kube-controller-manager
059d24ff6601        rancher/hyperkube:v1.14.3-rancher1                                                                                        "/opt/rke-tools/en..."   35 minutes ago      Up 35 minutes                           kube-apiserver
```
- `docker exec` into one of them, in my case it is `kubelet` 
-  check that zfs tools exist
```
# which zpool
/sbin/zpool
# which zfs
/sbin/zfs
```

**Comment by jiaqiluo** [comment]
The bug is reproduced in Rancher: v2.2.4

Do the same steps as the above comment provides, this time the default k8s version is `v1.13.5-rancher1-3` 

Result:
```
# which zfs
# which zpool
# zfs
bash: zfs: command not found
# zpool
bash: zpool: command not found
```
